orangecontrib/spectroscopy/preprocess/__init__.py

```diff
@@ -456,7 +456,6 @@ def __call__(self, data):
 class Interpolate(Preprocess):
     """
     Linear interpolation of the domain.
-
```

Comment:
Also, please avoid modifications like this when they do not touch your new feature. It makes history harder to read.

Reply:
Sorry Marko, this was an accidental pull. Please delete if possible

Reply:
I noticed, thanks, I just commented because these things are easier to solve when the code is still smaller. :)